features/step_definitions/activity_steps.rb

```diff
@@ -0,0 +1,3 @@
+Then (/^I start "([^\"]*)" activity with extras "([^\"]*)"$/) do |activity, extra|
+ system("adb shell am start -n " + "org.commcare.dalvik/org.commcare.activities." + activity + " " + extra)
```

Comment:
I thought we had decided it would be better to make this activity accessible via the developer options instead of launching it via adb, since adb calls can't run on AWS. Or has that limitation been addressed?

Reply:
I tested it on aws and faced no problems. I think we can run adb commands on aws provided they are part of step_definitions.

Reply:
Nice, that's good news